doc/user_guide/user_guide.md

```diff
@@ -188,56 +188,109 @@ Each algorithm should extend the `UDFQueryHandler` abstract class and then imple
 
 ### Concrete Example Using an Adhoc Implementation Within the UDF
 
-The example uses the module `builtins` and dynamically adds `ExampleQueryHandler` and `ExampleQueryHandlerFactory` to it.
+The example dynamically creates a python module `xyz` and adds classes `ExampleQueryHandler` and `ExampleQueryHandlerFactory` to it.
```

Comment:
I would give the module a more meaningful name than "xyz". If you were doing this manually you would probably call the module "example_query_handler.py" or "example_qh.py", or something like that.